packages/base/src/util/FocusableElements.ts

```diff
@@ -26,7 +28,7 @@ const isElemFocusable = (el: HTMLElement) => {
 	return el.hasAttribute("data-ui5-focus-redirect") || !isElementHidden(el);
```

Comment:
I think that getLastFocusableElement should have the same return type as getFirstFocusableElement.

Reply:
You are right. It is my typo, thanks !